src/wp-includes/blocks/latest-posts.php

```diff
@@ -151,6 +143,24 @@ function render_block_core_latest_posts( $attributes ) {
 
 			$trimmed_excerpt = get_the_excerpt( $post );
 
+			/*
+			 * Adds a "Read more" link with screen reader text.
+			 * [&hellip;] is the default excerpt ending from wp_trim_excerpt() in Core.
+			 */
+			if ( str_ends_with( $trimmed_excerpt, ' [&hellip;]' ) ) {
+				$excerpt_length = (int) apply_filters( 'excerpt_length', $block_core_latest_posts_excerpt_length );
+				if ( $excerpt_length <= $block_core_latest_posts_excerpt_length ) {
+					$trimmed_excerpt  = substr( $trimmed_excerpt, 0, -11 );
+					$trimmed_excerpt .= sprintf(
+						/* translators: 1: A URL to a post, 2: The static string "Read more", 3: The post title only visible to screen readers. */
+						__( '… <a href="%1$s" rel="noopener noreferrer">%2$s<span class="screen-reader-text">: %3$s</span></a>' ),
+						esc_url( $post_link ),
+						__( 'Read more' ),
+						esc_html( $title )
+					);
```

Comment:
Not sure if this is good from a i18n perspective. Specifically:
* Does it make sense for the `…` to be part of the (same) translation string?
* I think the "Read more" must not be outside of this main string, otherwise context is missing.
* Should we move the `a` tag outside the translation string and wrap it instead?
* The "Read more: {post title}" is a bit of a weird format text. Most core themes have solved this via something like `sprintf( __( 'Continue reading %s' ), $title )`.

Maybe something like this would be better:
```suggestion
					$trimmed_excerpt .= '… <a href="' . esc_url( $post_link ) . '" rel="noopener noreferrer">';
					$trimmed_excerpt .= sprintf(
						/* translators: %s: The post title only visible to screen readers */
						__( 'Continue reading <span class="screen-reader-text">%s</span>' ),
						esc_html( $title )
					);
					$trimmed_excerpt .= '</a>';
```

Reply:
Rule of thumb is to avoid string concatenation like in this suggestion.
The ellipsis should be part of the translatable string, which means keeping the `a` tag in there, too. But ses, the „Read more“ should be in there as well and not added via a placeholder.

Currently AFK, so I definitely do recommend checking similar instances in core themes and elsewhere, and asking @SergeyBiryukov or #polyglots if in doubt.

Reply:
These suggestions make sense to me, thank you! I've addressed them [here](https://github.com/WordPress/wordpress-develop/pull/5441/commits/ca4838232051de41ce9ee1267f3ec1306f99be42), but it would be great to get some feedback from more experienced i18n folks too.

Reply:
Discussion probably could continue on [GB55026](https://github.com/WordPress/gutenberg/issues/55026).

This is a pair of similar strings. "Read more" already has been visible on the front end since 6.3, so we may need to keep that exact text in the string.